server/services/transactionEventsController.go

```diff
@@ -58,6 +58,21 @@ func (controller *transactionEventsController) extractEventTransferValueOnly(tx
 	}, nil
 }
 
+func (controller *transactionEventsController) hasAnySignalError(tx *transaction.ApiTransactionResult) bool {
+	if !controller.hasEvents(tx) {
+		return false
+	}
+
+	for _, event := range tx.Logs.Events {
+		isSignalError := event.Identifier == transactionEventSignalError
```

Comment:
you can use this const from core `core.SignalErrorOperation`

Reply:
Thanks, fixed.